velox/docs/functions/spark/string.rst

```diff
@@ -108,6 +108,29 @@ String Functions
         SELECT find_in_set(NULL, ',123'); -- NULL
         SELECT find_in_set("abc", NULL); -- NULL
 
+.. spark:function:: initcap(string) -> varchar
+
+   Capitalizes the first letter of each word in a specified string.
+
+   The ``initcap`` function converts the first character of each word to uppercase
+   and all other characters in the word to lowercase. It supports UTF-8 multibyte
+   characters, up to four bytes per character.
+
+   A *word* is defined as a sequence of characters separated by whitespace. Only
+   space characters—such as space, tab, newline, carriage return, and form feed—
+   are considered word boundaries.
```

Comment:
It looks like Spark only consider white-space as delimiter. Would you double check?

https://github.com/apache/spark/blob/v3.5.1/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L1782